darshan-util/pydarshan/darshan/discover_darshan.py

```diff
@@ -201,7 +201,7 @@ def find_utils(ffi, libdutil):
         try:
             darshan_path = discover_darshan_pyinstaller()
             import glob
-            library_path = glob.glob(f'{darshan_path}/libdarshan-util*.so')[0]
+            library_path = glob.glob(f'{darshan_path}/libdarshan-util*.so*')[0]
```

Comment:
Not immediately clear what these changes to the glob here are trying to do. Could you provide some more details just so I better understand what they are fixing?

Reply:
The names of the shared object files changed slightly, they are now postfixed with .0.0.0. Without the wildcard for the glob pattern an empty list would be returned and the lib-darshan util discovery fails for this variant.